buildah_intro/buildah_intro.sh

```diff
@@ -5,6 +5,12 @@
 # for Buildah basic concepts and uses.  Also requires
 # the Docker and Podman packages to be installed.
 
+ 
+#Setting up some colors for helping read the demo output
+bold=$(tput bold)
+blue=$(tput setaf 4)
+reset=$(tput sgr0)
+
```

Comment:
If you don't want bold and/or blue, just comment out the corresponding line before a run.  This is one of @ipbabbles's tricks that I stole.